api/api.go

```diff
@@ -376,7 +384,61 @@ func (m *ApiService) handleValidatorOffchainStateByIndex(w http.ResponseWriter,
 	})
 }
 
+func (m *ApiService) handleValidatorRelayers(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	valPubKey := vars["valpubkey"]
+	if !IsValidPubkey(valPubKey) {
+		m.respondError(w, http.StatusInternalServerError, fmt.Sprintf("invalid validator pubkey format"))
+		return
+	}
+	var relays = []string{
+		"boost-relay.flashbots.net",
+		"bloxroute.max-profit.blxrbdn.com",
+		"bloxroute.ethical.blxrbdn.com",
+		"bloxroute.regulated.blxrbdn.com",
+		"builder-relay-mainnet.blocknative.com",
+		"relay.edennetwork.io",
+		"mainnet-relay.securerpc.com",
+		"relayooor.wtf",
+		"relay.ultrasound.money",
+		"agnostic-relay.net",
+		"aestus.live",
+		"builder-relay-goerli.flashbots.net",
+	}
+	var registeredRelays []string
+	//var registeredRelaysString bytes.Buffer
+	var regRelayString strings.Builder
```

Comment:
lets use `var regRelayString []strings`. See my other comment in `httpOkRelayersState`